src/auth/crud.py

```diff
@@ -20,7 +21,16 @@ async def create_user_session(db_session: AsyncSession, session_id: str, computi
         query = sqlalchemy.select(models.User).where(models.User.computing_id == computing_id)
         existing_user = (await db_session.scalars(query)).first()
         if existing_user is None:
-            # @geb; throw?
+            # log this strange case
+            _logger = logging.getLogger(__name__)
+            _logger.warning("User session {} exists for non-existent user {}!".format(session_id, computing_id))
```

Comment:
This is not important, but python now has f strings, like `f"User session {session_id} exists for non-existent user {computing_id}!"`, which is pretty nifty

Reply:
Oooo that is nifty!

Reply:
Lemme do that real quick for u